css/properties/shape-outside.json

```diff
@@ -467,39 +311,13 @@
               "edge": {
                 "version_added": false
               },
-              "firefox": [
-                {
-                  "version_added": "62",
-                  "version_removed": "70"
-                },
-                {
-                  "version_added": "61",
-                  "version_removed": "62",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "layout.css.shape-outside.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                }
-              ],
-              "firefox_android": [
-                {
-                  "version_added": "62"
-                },
-                {
-                  "version_added": "61",
-                  "version_removed": "62",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "layout.css.shape-outside.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                }
-              ],
+              "firefox": {
+                "version_added": "62",
+                "version_removed": "70"
+              },
+              "firefox_android": {
+                "version_added": "62"
```

Comment:
What's going on here? Was this really removed on desktop but remains on Android?

Reply:
Probably another issue due to the pause on FxA while the new version was in development.  Fixing!